Profesja/Include/Profesja.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include <iostream>
+#include <vector>
+#include <string>
+
+struct Player
+{
+  unsigned int currentHp = 20;
+  unsigned int power = 1;

```

Comment:
Za co ma odpowiadać `power`?

+ uint

Reply:
chcieliśmy ograniczyć ilość zmiennych więc power odpowiada jednocześnie za moc leczenia i zadawania obrażeń

Reply:
Na razie wrzucić jako zmienna lokalna funkcji useUlti. Potem gdybyśmy chcieli modyfikować ją co lvl to zależnie od tego przeniesiemy lub będziemy mieli mnożnik od lvl'a.